internal/cli/pipeline_run.go

```diff
@@ -125,20 +125,33 @@ func (c *PipelineRunCommand) Run(args []string) int {
 
 		successful := steps
 		for _, jobId := range resp.AllJobIds {
-			app.UI.Output("Executing Step %q", resp.JobMap[jobId].Step, terminal.WithHeaderStyle())
+			job, err := c.project.Client().GetJob(c.Ctx, &pb.GetJobRequest{
+				JobId: jobId,
+			})
+			if err != nil {
+				return err
+			}
+			ws := "default-nil"
```

Comment:
Oh should this be `default`?

Reply:
Good catch Brian, thanks, fixed in 729f4fa9b